test/fixtures/extract-css/nuxt.config.js

```diff
@@ -12,6 +12,9 @@ export default {
           grid: true
         }
       }
+    },
+    extend(config) {
```

Comment:
What’s this for?

Reply:
Whoops. Leftover from testing the other PR I guess. :see_no_evil: